CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased]
 
+- Added support for configuration of resources and node selectors with node tags
```

Comment:
with node tags -> with [Kedro node tags](https://kedro.readthedocs.io/en/stable/nodes_and_pipelines/nodes.html#how-to-tag-a-node)

(with link)